src/main/scala/vectorpipe/osm/package.scala

```diff
@@ -184,7 +185,12 @@ package object osm {
     *
     * @group conversion
     */
-  def features(ns: RDD[(Long, Node)], ws: RDD[(Long, Way)], rs: RDD[(Long, Relation)]): Features = {
-    PlanetHistory.features(ns, ws)
+  def features(
```

Comment:
A note in the docstrings here about the various Storage Levels might be handy, unless that's something that's better addressed in the Microsite docs.